Formula/crystal-lang.rb

```diff
@@ -55,6 +70,8 @@ def install
     macho = MachO.open("#{buildpath}/boot/embedded/bin/crystal")
     macho.change_dylib("/usr/local/opt/libevent/lib/libevent-2.0.5.dylib",
                        "#{buildpath}/vendor/libevent/lib/libevent-2.0.5.dylib")
+    macho.change_dylib("/usr/local/opt/bdw-gc/lib/libgc.1.dylib",
```

Comment:
Nope. The versions may not match, which was the reason for the corresponding libevent change, not the prefix location.